data_pipelines_cli/cli_commands/compile.py

```diff
@@ -60,10 +87,10 @@ def _docker_build(docker_args: DockerArgs) -> None:
 def _dbt_compile(env: str) -> None:
     generate_profiles_yml(env)
     echo_info("Running dbt commands:")
-    dbt(("deps",), env)
-    dbt(("compile",), env)
-    dbt(("docs", "generate"), env)
-    dbt(("source", "freshness"), env)
+    dbt(("deps",), env, generate_profiles=True)
+    dbt(("compile",), env, generate_profiles=False)
+    dbt(("docs", "generate"), env, generate_profiles=False)
+    dbt(("source", "freshness"), env, generate_profiles=False)
```

Comment:
I think we should simplify it by calling method generate_profile before dbt method and either pass it's path to dbt or build the path inside.